websites/P/Physics Wallah/presence.ts

```diff
@@ -0,0 +1,132 @@
+const presence = new Presence({
+		clientId: "1134044987277975616",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let mediaTimestamps: [number, number];
+
+const enum Assets {
+	HomePage = "https://i.imgur.com/pIrO5z2.png",
+	Scrolling = "https://i.imgur.com/klh7wd3.png",
+	Logo = "https://i.imgur.com/OxQKsGm.png",
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname, href, search } = document.location;
+
+	if (pathname === "/") {
+		presenceData.details = "Home";
+		presenceData.state = "Browsing...";
+		presenceData.smallImageKey = Assets.HomePage;
+		presenceData.smallImageText = "Browsing Home Page";
+	}
+
+	if (pathname.startsWith("/study")) {
+		presenceData.details = "Browsing...";
+		presenceData.state = "In website";
+		presenceData.smallImageKey = Assets.Scrolling;
+		presenceData.smallImageText = "Browsing the website";
+
+		if (pathname.endsWith("/my-batches")) {
+			presenceData.details = "Studying...";
+			presenceData.state = "My Batches";
+			presenceData.smallImageKey = Assets.Reading;
+			presenceData.smallImageText = "Studying";
+		}
+
+		if (pathname.includes("batch-overview")) {
+			presenceData.details = "Studying...";
+			presenceData.state = `Viewing ${
```

Comment:
To avoid confusion, it may be better to put separate paths as part of an `else if`, rather than having just `if`.
```ts
if (pathname === "/") {
  // stuff
} else if (pathname.includes("/otherpath")) {
  // etc
} else if (something) {
  // ...
}
```